zebrad/src/components/sync/tests.rs

```diff
@@ -1,4 +1,5 @@
 //! Syncer tests
 
+mod eos;
```

Comment:
This tests only work when I run them separated from others. When running the whole suite they fail with :

```
Could not set global tracing subscriber: SetGlobalDefaultError("a global default trace dispatcher has already been set")
```

There is a conflict with `tracing_test` and already subscribed code. I tried different alternatives today here, none of them solved it. I might go in a different way than capturing the logs in the test.